### DISCUSSION TITLE
Human being

### DISCUSSION THREAD

**Discussion by goforward10** [community/community#124676] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi Need to find the old password for my

**Comment by LiteBrite82** [comment, marked as answer]
Hi @goforward10 ! 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

I am also going to remove your personal email from your post for your safety in accordance with our community guidelines.
